json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -326,6 +327,11 @@ public function get_settings_response() {
 
 				$api_cache = $is_jetpack ? (bool) get_option( 'jetpack_api_cache_enabled' ) : true;
 
+				$climatestrike_option = get_option( 'digital-climate-strike' );
```

Comment:
May be too late now because it is on dotcom, but should this be prefixed with `jetpack_`?

Reply:
Yeah, I wouldn't worry too much about it since we'll most likely remove it very soon. 

We may actually end up removing the option before the next release; let's wait before we merge this.